UserSearcArrayList.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class UserSearcArrayList {
+
+	ArrayList<Integer> userarr;
+	ArrayList<String> emailarr;
```

Comment:
We are creating Lists here but name of the variables are still using array keyword